platforms/r3-corda-ent/charts/cenm-idman/templates/service.yaml

```diff
@@ -4,79 +4,42 @@
 #  SPDX-License-Identifier: Apache-2.0
 ##############################################################################################
 
+---
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Values.nodeName }}
-  namespace: {{ .Values.metadata.namespace }}
+  name: {{ include "idman.name" . }}
+  namespace: {{ .Release.Namespace }}
   labels:
-    run: {{ .Values.nodeName }}
-    app.kubernetes.io/name: {{ .Values.nodeName }}
-    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: idman-service
+    app.kubernetes.io/component: idman
+    app.kubernetes.io/part-of: {{ include "idman.fullname" . }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/namespace: {{ .Release.Namespace }}
+    app.kubernetes.io/release: {{ .Release.Name }}
 spec:
+  type: ClusterIP
   selector:
-    app: {{ .Values.nodeName }}
-# we need Health Check node port set to get rid of logs pollution
-{{- if (.Values.healthCheck.nodePort) }}
-  healthCheckNodePort: {{ .Values.healthCheck.nodePort }}
-{{- end }}
+    app.kubernetes.io/name: idman-statefulset
+    app.kubernetes.io/component: idman
+    app.kubernetes.io/part-of: {{ include "idman.fullname" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/namespace: {{ .Release.Namespace }}
+    app.kubernetes.io/release: {{ .Release.Name }}
   ports:
-    - port: {{ .Values.service.internal.port }}
-      targetPort: {{ .Values.service.internal.port }}
-      protocol: TCP
-      name: issuance
-    - port: {{ .Values.service.revocation.port }}
-      targetPort: {{ .Values.service.revocation.port }}
-      protocol: TCP
-      name: revocation
-    - port: {{ .Values.service.external.port }}
-      targetPort: {{ .Values.service.external.port }}
-      protocol: TCP
-      name: main
-    - port: {{ .Values.service.adminListener.port }}
-      targetPort: {{ .Values.service.adminListener.port }}
-      protocol: TCP
-      name: adminlistener
-{{ if $.Values.ambassador }}
----
-apiVersion: getambassador.io/v3alpha1
```

Comment:
Is ambassador not needed?

Reply:
this PR is for no proxy setup only, will need ambassador related code and is WIP with issue #2540